### PR TITLE
fetching image fixed

### PR DESCRIPTION
**Pull request by prathmeshkhandelwal1**
@iiitv/project-maintainers Can you review this?

### CODE REVIEW

File: _data/author.yml [hunk 1]
```diff
@@ -4,5 +4,5 @@ aviaryan:
     name: Avi Aryan
     twitter: aviaryan123
     tagline: Competitive programmer, Loves to build new things, Anime addict, Will travel the world some day
-    img: http://aviaryan.in/images/profile.png
+    img: https://github.com/aviaryan
```
Comment (aashutoshrathi) [inline _data/author.yml]: `https://github.com/aviaryan.png`

File: _data/author.yml [hunk 2]
```diff
@@ -4,5 +4,5 @@ aviaryan:
...
     link: http://aviaryan.in
```
Comment (aashutoshrathi) [inline _data/author.yml]: Newline at the end please

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Please check before pushing

File: _data/author.yml (hunk 2 above)
Comment (anshumanv) [inline _data/author.yml]:
```suggestion
    link: https://aviaryan.in
```

Comment (anshumanv) [reply _data/author.yml]: Change this before @aashutoshrathi is triggered 😄

**Comment by prathmeshkhandelwal1** [comment]
@aashutoshrathi sorry sir, silly mistake in copy paste :(

**Comment by prathmeshkhandelwal1** [comment]
@aashutoshrathi will do in a minute.

**Review by AmanRaj1608 — APPROVED** [review]

File: _data/author.yml [hunk 3]
```diff
@@ -4,5 +4,5 @@ aviaryan:
     name: Avi Aryan
     twitter: aviaryan123
     tagline: Competitive programmer, Loves to build new things, Anime addict, Will travel the world some day
-    img: http://aviaryan.in/images/profile.png
-    link: http://aviaryan.in
\ No newline at end of file
+    link: https://aviaryan.in
+    img: https://github.com/aviaryan.png
```
Comment (aashutoshrathi) [inline _data/author.yml]: Can you add a new empty line at the end?

File: _data/author.yml [hunk 4]
```diff
@@ -5,4 +5,5 @@ aviaryan:
     twitter: aviaryan123
     tagline: Competitive programmer, Loves to build new things, Anime addict, Will travel the world some day
     link: https://aviaryan.in
-    img: https://github.com/aviaryan.png
\ No newline at end of file
+    img: https://github.com/aviaryan.png
+
```
Comment (aashutoshrathi) [inline _data/author.yml]: This line contains whitespaces 😞

Comment (prathmeshkhandelwal1) [reply _data/author.yml]: tenor

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**